components/nimbus/src/persistence.rs

```diff
@@ -442,6 +559,503 @@ mod tests {
+                "id":"branch-feature-feature-id-empty", // change when cloning
+                "last_modified":1_602_197_324_372i64
+            }),
+            json!({
+                "schemaVersion": "1.0.0",
+                "slug": "branch-feature-value-missing", // change when cloning // XXX verify that we really to discard these and that there aren't any live experiments with have this problem.  If so, clean up all remaining experiments in this list
+                "endDate": null,
+                "featureIds": ["ggg"], // change when cloning
+                "branches":[
+                    {
+                        "slug": "control",
+                        "ratio": 1,
+                        "feature": {
+                            "featureId": "ggg", // change when cloning
+                            "enabled": false
+                        }
+                    },
+                    {
+                        "slug": "treatment",
+                        "ratio":1,
+                        "feature": {
+                            "featureId": "ggg", // change when cloning
+                            "enabled": true
+                        }
+                    }
+                ],
+                "channel": "nightly",
+                "probeSets":[],
+                "startDate":null,
+                "appName": "fenix",
+                "appId": "org.mozilla.fenix",
+                "bucketConfig":{
+                    // Setup to enroll everyone by default.
+                    "count":10_000,
+                    "start":0,
+                    "total":10_000,
+                    "namespace":"branch_feature_value_missing", // change when cloning
+                    "randomizationUnit":"nimbus_id"
+                },
+                "userFacingName":"Diagnostic test experiment",
+                "referenceBranch":"control",
+                "isEnrollmentPaused":false,
+                "proposedEnrollment":7,
+                "userFacingDescription":"This is a test experiment for diagnostic purposes.",
+                "id":"branch_feature_value_missing", // change when cloning
+                "last_modified":1_602_197_324_372i64
+            }),
+        ]
+    }
+
+    // fn get_valid_feature_enrollments() -> Vec<serde_json::Value> {
+    //     vec![json!(
+    //         {
+    //             "slug": "secure-gold",
+    //             "status":
+    //                 {
+    //                     "Enrolled":
+    //                         {
+    //                             "enrollment_id": "801ee64b-0b1b-44a7-be47-5f1b5c189083", // change on cloning
+    //                             "reason": "Qualified",
+    //                             "branch": "control",
+    //                             "feature_id": "about_welcome" // change on cloning
+    //                         }
+    //                     }
+    //                 }
+    //     )]
+    // }
```

Comment:
nit: Let's get rid of this if it isn't needed

Reply:
Similarly, this will all morphed into something that is used or dropped before landing.